vpu/generic/vpuGeneric/Dockerfile

```diff
@@ -0,0 +1,222 @@
+ARG BASE_VERSION=3
+
+##
+# Board architecture
+# arm or arm64
+##
+ARG IMAGE_ARCH=
+
+##
+# Board GPU vendor prefix
+##
+ARG GPU=
+
+##
+# SoM
+##
+#ARG MACHINE=verdin-imx8mm
+ARG MACHINE=verdin-imx8mp
```

Comment:
Is there a way to change this programmatically? e.g. based on default default connected device

Reply:
@microhobby probably know

Reply:
Noted, thanks!